src/App.tsx

```diff
@@ -1,25 +1,48 @@
-import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import { Instrument } from './features/instrument';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { AppBar } from '@mui/material';
+import { teal, common } from '@mui/material/colors';
+
+const theme = createTheme({
+  palette: {
+    mode: 'dark',
+    primary: {
+      main: common.white
+    },
+    secondary: teal,
+  },
+  components: {
+    MuiButton: {
+      styleOverrides: {
```

Comment:
🤮

Does that look ok for you? 

Context: It's my first try to use some UI lib for React other than Polaris. I wanted to use some material design lib, `mui` appeared to be very popular in React world.
I found it so terrible to customize current components. And I'm using only buttons :D 
**It's ok if you are not familiar and don't want dig into, ignore this comment, but if you are - is customization supposed to be like that? That's all fine?** I'm considering to have a look into other libs later

It's so odd to set stylings via js/ts. But that's the only working and semi-okay-looking way I found, other options: 
-set styles in component's using `!important` everywhere
-set inline styles using still js/ts on every component using magic `sx` prop, couldn't avoid using once

I couldn't make to work:
-[overriding styles with class names](https://mui.com/customization/how-to-customize/#overriding-styles-with-class-names), not sure that's to be better due to weird-looking naming, TBH
-[other deprecated ways in which are still in docs](https://mui.com/styles/advanced/)